src/main/resources/static/assets/css/custom.css

```
@@ -357,3 +356,7 @@ p.normal {
 input[name$="Address2"] {
   max-width: 6em;
 }
+
+.grey-darkest {
+  color: var(--gray-darkest)
+}
```

Please name this [similar to our other color utility classes](https://github.com/codeforamerica/il-gcc-form-flow/blob/0fae523eb80157b62d292509fa441e125c4aeb11/src/main/resources/static/assets/css/custom.css#L103):

```suggestion
.color--gray-darkest {
  color: var(--gray-darkest);
}

```